magento/magento2
# Products in cart report error when we have grouped or bundle product

## Description
Changed inner join with left join for prices because bundle and grouped products don't have prices and report generates error if we have them in cart.

## Fixed Issues
magento/magento2#12079:Products in cart report error when we have grouped or bundle product

### Manual testing scenarios
1. Add a grouped or a bundle product to cart.
2. Go to admin->Reports->Marketing->Products in cart and loop through pages or export to csv/excel
3. Looping through pages or exporting should no longer generate error 

### Contribution checklist
